connection.go

```diff
@@ -758,6 +759,14 @@ func (sc *snowflakeConn) buildRowsForRunningQuery(ctx context.Context, qid strin
 	return rows, err
 }
 
+func (sc *snowflakeConn) setupOCSPPrivatelink(app string, host string) error {
```

Comment:
This change is not enough. It's not only the cache server URL that changes but also the actual OCSP responder URL that changes as well. 

`ocspRetryHost := fmt.Sprintf("http://ocsp.%v/retry/\%v/\%v", host)`
`if err: = os.Setenv("ocspRetryURLEnv, ocspRetryHost); err != nil {`
`            return err`
`}`

The above change would export an environment variable `ocspRetryURLEnv` with the value something like : 
`http://ocsp.abc.privatelink.snowflakecomputing.com/retry/%v/%v`

Once this is done, in ocsp.go within `getRevocationStatus` you would want to change line `489` to first check if `ocspRetryURLEnv` is set. If it is, you should extract that value and create retry URL from it like so : 

` hostname := fmt.Sprintf(ocspRetryURLEnv, url.Parse(ocspHost).Hostname(), ocspReq)`

this would set the ocspRetryURL to something like : 
`http://ocsp.abc.privatelink.snowflakecomputing.com/retry/msocsp.com/<Base64 OCSP request>`